torch/testing/_internal/common_utils.py

```diff
@@ -527,6 +527,9 @@ def run_unittest_help(argv):
     # infer flags based on the default settings
     GRAPH_EXECUTOR = cppProfilingFlagsToProfilingMode()
 
+RERUN_DISABLED_TESTS = args.rerun_disabled_tests
+# Rerun disabled tests many more times to make sure that they are not flaky anymore
+MAX_NUM_RETRIES = 3 if RERUN_DISABLED_TESTS else 50
```

Comment:
Let's see how long it takes to rerun the failed test 50 times.  One of the annoying thing I observe is that distributed tests still take a while to run even that most tests are skipped because I believe spawning new sub-processes is done before `check_if_enable`. So even if `check_if_enable` says no, the cost of spinning up new sub-processes already applies

Reply:
i think the if statement should be not RERUN_DISABLED_TESTS? or flip the 3 and the 50

Reply:
lol, you're right, it should be the other way around.  Silly me!